code/modules/antagonists/changeling/powers/gaseous_pores.dm

```diff
@@ -0,0 +1,23 @@
+/datum/action/changeling/gaseous_pores
+	name = "Gaseous Pores"
+	desc = "Our skins bursts, releasing noxious gases to confuse opponents and cover our retreat."
+	helptext = "We are still affected by the gases we emit, internals are recommended.Functions while unconscious."
+	button_icon_state = "smoke"
+	chemical_cost = 35
+	dna_cost = 2
+	req_stat = UNCONSCIOUS
+	var/silent = FALSE
+
+/datum/action/changeling/gaseous_pores/sting_action(mob/user)
+	..()
+	var/turf = get_turf(user)
+	var/smoke_type = /obj/effect/particle_effect/smoke/sleeping
+	var/range = 4
+	var/datum/effect_system/smoke_spread/sleeping/smoke = new(turf)
+	smoke.effect_type = smoke_type
+	smoke.set_up(range, turf)
+	smoke.start()
+	if(!silent)
```

Comment:
is there any way to get `silent = TRUE` ? If not, why not just remove it?

Reply:
Yeah, fixed